src/main/java/com/example/petbuddybackend/service/block/BlockService.java

```diff
@@ -34,14 +33,14 @@ public class BlockService {
     private final UserMapper userMapper = UserMapper.INSTANCE;
     private final ApplicationEventPublisher eventPublisher;
 
-    public Page<AccountDataDTO> getUsersBlockedByUserSortedByBlockedUsername(String username, Pageable pageable) {
-        Pageable sortedPageable = PagingUtils.sortedBy(
-                pageable, BLOCKED_EMAIL_PROPERTY_NAME, Sort.Direction.ASC);
-
-        return blockRepository.findByBlockerEmail(username, sortedPageable)
+    public List<AccountDataDTO> getUsersBlockedByUserSortedByBlockedUsername(String username) {
+        return blockRepository.findByBlockerEmail(username)
+                .stream()
                 .map(Block::getBlocked)
                 .map(userService::renewProfilePicture)
-                .map(userMapper::mapToAccountDataDTO);
+                .map(userMapper::mapToAccountDataDTO)
+                .sorted(Comparator.comparing(AccountDataDTO::email))
+                .toList();
```

Comment:
W sumie jakby dodać coś w stylu `findByBlockerEmailSortByBlockerEmail` to powinno być szybsze i nie będzie trzeba tej linijki dodawać

`.sorted(Comparator.comparing(AccountDataDTO::email))`

Reply:
rzeczywiscie